src/Components/Dialogs/dialogs.jsx

```diff
@@ -3,20 +3,23 @@ import React from 'react';
 import GetPerson from './GetPerson/getPerson';
 import GetItem from './GetMessage/getMessage';
 
-import { messagesData, dialogsData } from '../../core/constants';
+import ROUTES from '../../constants';
 
 import style from './dialogs.module.css';
 
 
-const Dialogs = () => (
-  <div className={style.dialogs}>
-    <div className={style.dialogsItems}>
-      <GetPerson items={dialogsData} />
+const Dialogs = (props) => {
+  const { dialogsData, messagesData } = props.state;
+  return (
+    <div className={style.dialogs}>
+      <div className={style.dialogsItems}>
+        <GetPerson items={dialogsData} ROUTES={ROUTES} />
```

Comment:
Consider NOT naming the props in uppercase.

Reply:
Yes.sure I made a mistake